pom.xml

```diff
@@ -6,7 +6,6 @@
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
         <version>3.3.3</version>
-        <relativePath/>
```

Comment:
Why did you remove this? You should change it back.

Reply:
@MateuszRostek Sorry this was one of many steps I took trying to solve the issues I had with mvn clean package not passing without giving me too much info on what was wrong... But all sorted. Will correct this